### PR TITLE
updated caption

### PR DESCRIPTION
**Pull request by ughani900**

### CODE REVIEW

File: _slides/12-ughani900.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption Changed this
```
Comment (Duc-Vo) [inline _slides/12-ughani900.md]: Looks good to me.

Comment (Duc-Vo) [inline _slides/12-ughani900.md]:
```suggestion
Welcome to Robotocat's Slide.
```

Comment (h3rryqle) [inline _slides/12-ughani900.md]: Looking good

**Review by h3rryqle — APPROVED** [review]

**Review by Duc-Vo — APPROVED** [review]
Looks good to me! Ready to deploy!
👍 🚀 🚢 ✅

**Comment by ughani900** [comment]
👍

**Comment by ughani900** [comment]
merge

**Pull request closed without merging**